src/main/php/inject/Injector.class.php

```diff
@@ -235,7 +226,7 @@ public function get($type, $name= null) {
   /**
    * Retrieve args for a given routine.
    *
-   * @param  lang.reflect.Routine $routine
+   * @param  lang.reflection.Routine $routine
```

Comment:
This is a BC break and will require a major version release (6.0.0 at the time of writing).